client/hotrod/src/test/java/org/infinispan/hotrod/test/AbstractSingleHotRodServerTest.java

```diff
@@ -0,0 +1,48 @@
+package org.infinispan.hotrod.test;
+
+import static org.infinispan.hotrod.HotRodServerExtension.builder;
+
+import java.util.concurrent.TimeUnit;
+
+import org.infinispan.api.async.AsyncCache;
+import org.infinispan.api.async.AsyncContainer;
+import org.infinispan.commons.util.concurrent.CompletableFutures;
+import org.infinispan.hotrod.HotRodServerExtension;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+/**
+ * @since 14.0
+ */
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public abstract class AbstractSingleHotRodServerTest<K, V> {
```

Comment:
We should probably rename this to Async somehow as it contains the AsyncCache.

Reply:
Renamed to a mouthful of a name.